### ISSUE TITLE
Use inspect.getmodule to get an object's module

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5866]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#5866|
|Type|enhancement|
|Created|2012-08-14 15:17:23Z|

As per the (removed) comment, `inspect.getmodule` works in 2.6+, so use it.

Attachments:

* [getmodule.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/8d0/8d0fcd4da5dca9b03ff24e744e411f1c9bedcbc4/8ea42680bff635fa69468b6838a33722bda97291.patch) (625 bytes) - added by Julian on 2012-08-14 15:17:31Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5866 5866
type__enhancement enhancement
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__invalid invalid
component__core core
keywords__ 
time__1344957443000000 1344957443000000
changetime__1347211958000000 1347211958000000
version__None None
owner__Julian Julian

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve set owner to @Julian|
|-|-|

I'm not sure what's the policy here, but the fix is not in 2.6.0. Considering it doesn't seem to be major, I'd leave it like it is for now.

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_2"></a>|@Julian set status to `closed`|
|-|-|

Ouch. Thanks for catching that. Sounds good.
